### PR TITLE
Feature/fast slam

### PR DESCRIPTION
**Pull request by verbal-noun**
Finished particle predicting code and created a Jupyter notebook for other teammates to build upon. 

The .py also contains commenting guidelines for others to maintain for this file. ( Because multiple formats of commenting might be confusing)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (JmcRobbie) [inline .vscode/settings.json]: Delete this file

File: nova_rover_demos/SLAM/.ipynb_checkpoints/FastSLAM Demonstration-checkpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,752 @@
+{
```
Comment (JmcRobbie) [inline nova_rover_demos/SLAM/.ipynb_checkpoints/FastSLAM Demonstration-checkpoint.ipynb]: This file as well should be removed from the pull request

**Review by JmcRobbie — CHANGES_REQUESTED** [review]

**Comment by JmcRobbie** [comment]
Looks great, delete the unnecessary files and then we should be good to merge!

**Review by JmcRobbie — APPROVED** [review]
Nice job

**Comment by verbal-noun** [comment]
No actual changes were made since the last approval. Hence I will merge the pull request.

**Pull request merged**